qemu/tests/cfg/numa_opts.cfg

```diff
@@ -101,11 +101,17 @@
             numa_expected = "0-63"
         - nodes.128:
             no aarch64
+            x86_64:
+                no RHEL.9.2
             type = numa_maxnodes
             numa_nodes = 128
-            mem_fixed = 4G
-            vm_mem_minimum = 4G
-            node_size = 32M
+            mem_fixed = 16G
+            vm_mem_minimum = 16G
+            node_size = 128M
+            RHEL.7, RHEL.8, RHEL.9.0, RHEL.9.1:
```

Comment:
Why we don't have RHEL.9.4.0 and it seems RHEL.9.1 is EOF. For RHEL 9.2.0 is EUS version, which is more important.

Reply:
@MiriamDeng RHEL.9.3.0, RHEL.9.4.0 (and so on in the future) will have the default configuration from lines 108 to 110.
For the case of RHEL.9.2.0, please check the last comment in 1589

Reply:
OK, got it, thanks ! Regrading to RHEL 9.2, do you think we can add #bugid to this cfg  ? Thanks.

Reply:
@MiriamDeng yes, I think so :wink:

Reply:
Done!